tests/write_font_test.py

```diff
@@ -93,6 +93,12 @@ def test_keep_glyph_names(svgs, color_format, keep_glyph_names):
             "clocks_picosvg.ttx",
             {"color_format": "picosvg"},
         ),
+        # passing a tiny --reuse_tolerance effectively disables shape reuse
```

Comment:
Maybe we should have a value that actually disables it, such as -1

Reply:
Done